rb/spec/integration/selenium/webdriver/manager_spec.rb

```diff
@@ -64,6 +64,19 @@ module WebDriver
           expect(cookies.first[:value]).to eq('bar')
         end
 
+        it 'should add sameSite cookie with attribute Strict', only: {browser: :chrome} do
+          driver.navigate.to url_for('xhtmlTest.html')
+          driver.manage.add_cookie name: 'foo', value: 'bar', sameSite: 'Strict'
```

Comment:
I believe we should use snake case for paramater to make the code more Rubyish:

```suggestion
          driver.manage.add_cookie name: 'foo', value: 'bar', same_site: 'Strict'
```